platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MockAppender.java

```diff
@@ -38,15 +37,15 @@
 public class MockAppender extends AbstractAppender {
     private final List<String> messages = new CopyOnWriteArrayList<>();
 
-    public MockAppender(@NonNull final String name) {
+    public MockAppender(final String name) {
         super("MockAppender-" + name, null, null, true, null);
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
-    public void append(@NonNull final LogEvent event) {
+    public void append(final LogEvent event) {
```

Comment:
Minor: while `@NonNull` may not be my personal favorite annotation, it's still unclear why this fix removes them from here. From the code logic perspective, they kind of make sense here. And this change doesn't seem to be related to the "unit test modules removal". So I'm not sure what this diff is doing here.